exercises/binary/src/test/java/BinaryTest.java

```diff
@@ -12,10 +12,11 @@
 @RunWith(Parameterized.class)
 public class BinaryTest {
 
-    private String input;
-    private int expectedOutput;
+    private String binaryNumberAsString;
+    private int decimalNumber;
 
-    @Parameters
+    @Parameters(name = "{index}: expected result {1} when converting binary number " +
```

Comment:
"The fewer words written, the more of them read."  Do you think we can trim up this sentence?